src/clojure_tic_tac_toe/board.clj

```diff
@@ -0,0 +1,18 @@
+(ns clojure-tic-tac-toe.board)
+
+(defn add-move
+  [board move player]
+  (update board player conj move))
+
+(defn filter-out-player-moves
```

Comment:
should this be a private function?

Reply:
There is probably a generic higher order function that you wish you had, but had to write this helper instead.  Ideally, you should write that generic higher order function in some sort of helper namespace.  It would be public in that namespace and can be tested there.  You can then require and call it here without it leaking into the public scope of `board`.

Reply:
Moved the function to the `helper` namespace